helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -135,3 +135,39 @@ spec:
         severity: page
         team: atlas
         topic: observability
+  - name: loki.compactor
+    rules:
+    - alert: LokiCompactorFailedCompaction
+      annotations:
+        dashboard: loki-retention/loki-retention
+        description: 'Loki compactor has been failing compactions for more than 2 hours since last compaction.'
+        opsrecipe: loki#lokicompactorfailedcompaction
+      # This alert checks if Loki's the last successful compaction run is older than 2 hours
+      expr: (time() - (loki_boltdb_shipper_compact_tables_operation_last_successful_run_timestamp_seconds > 0) > 60 * 60 * 2)
+      for: 1h
+      labels:
+        area: platform
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: atlas
+        topic: observability
+    - alert: LokiCompactorFailedCompaction
```

Comment:
Both alerts have the same name. Does it work? Won't we get into trouble at some point?

Reply:
There's no reason why 2 alerts can have the same name like https://github.com/grafana/mimir/blob/2a0738f7b187664194a3af42c9b700a79d27d179/operations/mimir-mixin-compiled/alerts.yaml#L876C18-L876C63 and https://github.com/grafana/mimir/blob/2a0738f7b187664194a3af42c9b700a79d27d179/operations/mimir-mixin-compiled/alerts.yaml#L890 :)

Reply:
Oh, I didn't know. We probably could have split some other alerts like that, rather than doing some convoluted `or` list of queries.